components/molecules/SectionsTree.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import type { Story } from "@storybook/react";
+import SectionsTree from "./SectionsTree";
+import TreeView from "@material-ui/lab/TreeView";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import ChevronRightIcon from "@material-ui/icons/ChevronRight";
+import { sections } from "$samples";
+
+export default {
+  title: "molecules/SectionsTree",
+  component: SectionsTree,
+  argTypes: { selectable: { controls: "boolean" } },
+};
```

Comment:
プロパティ名が適切でないため、Controls、Docs等でBooleanのスイッチが表示されていませんでした

```suggestion
  component: SectionsTree,
  argTypes: { selectable: { control: "boolean" } },
};
```